topic-operator/src/test/java/io/strimzi/operator/topic/Utils.java

```diff
@@ -49,4 +52,18 @@ public static TopicMetadata getTopicMetadata(Topic kubeTopic) {
                 partitions), new Config(configs));
     }
 
+    public static String getLatestKafkaVersion() {
```

Comment:
Wouldn't it be better to move `io.strimzi.operator.cluster.KafkaVersionTestUtils` to common, than reinvent similar functionality?